vercel.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://openapi.vercel.sh/vercel.json",
+	"git": {
+		"deploymentEnabled": {
+			"main": false,
+			"master": false
+		}
+	},
+	"github": {
+		"silent": false,
+		"autoJobCancelation": true
+	},
+	"ignoreCommand": "git diff --quiet HEAD^ HEAD vercel.json ./package.json ./packages/contracts/package.json ./packages/contracts/contracts ./packages/DApp",
```

Comment:
no required, in dashboard – but for reviewability

Reply:
Why does it have `./packages/DApp` in the path?

Reply:
the path is for changes in files which the build is interested in

Reply:
if it's not quite, $? returns 1

Reply:
> If, however, a code "1" or greater is returned, then a new deployment will be built. – https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel